bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/observability/ObservabilityConstants.java

```diff
@@ -45,7 +45,9 @@ private ObservabilityConstants() {
     // Runtime related tags
     public static final String TAG_KEY_ERROR = "error";     // Standard boolean error tag
     public static final String TAG_KEY_ENTRYPOINT_FUNCTION_MODULE = "entrypoint.function.module";
-    public static final String TAG_KEY_ENTRYPOINT_FUNCTION_POSITION = "entrypoint.function.position";
+    public static final String TAG_KEY_ENTRYPOINT_OBJECT_NAME = "entrypoint.object.name";
+    public static final String TAG_KEY_ENTRYPOINT_FUNCTION_NAME = "entrypoint.function.name";
+    public static final String TAG_KEY_ENTRYPOINT_ACCESSOR_NAME = "entrypoint.accessor.name";
```

Comment:
Shall we change this to be consistent with `src.resource.accessor` tag ?
```suggestion
    public static final String TAG_KEY_ENTRYPOINT_RESOURCE_ACCESSOR = "entrypoint.resource.accessor";
```

Reply:
Fixed in cc06a1391b7e2bbe4071119cfa474801cee21e57

Reply:
Fixed in deebe9e0fbd82b278332e377842f2c99023dae07